### ISSUE TITLE
IL071 MOD - T First

### ISSUE THREAD

**Issue by scronaction** [prjct92eh2/Prjct92eh2-ST#2]
I've been trying to display temperature as the main tile instead of motion, to no avail. I found 2-3 discussions but not solutions.

Jimmy - Do you think you can modify the handler to display T first, as the primary?

**Comment by Automated-House** [comment]
I haven't messed with the new CLI for modifying the display in the new app at all :( Unfortunately it's not as easy for the new app as it was for Classic.

**Comment by scronaction** [comment]
Yeah, I miss the classic. The thing is, smarttiles can do it without any code changes. I feel like there's an easy but well hidden solution, maybe a new groovy function or something.

It's an issue with a bunch of other multi capability sensors, not limited to iris. 

Anyhow, it was worth a shot. Thanks for the response.

PS: For others who may be reading this, I was able to achieve the same functionality by creating a simulated zigbee temperature sensor, pulling the data from the sensor. But, it cannot be used for much else. I have never been able to create a reliable 2 way communication thru a simulated sensor.
